### PR TITLE
Added Haxe setup

### PR DESCRIPTION
**Pull request by IAmCheeseman**

### CODE REVIEW

File: templates/setupHaxeProject.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - name: "Create .gitignore and its contents"
+    run: touch .gitignore ;
+            echo 'bin' >> .gitignore ;
+  - name: "Create Main.hx and its contents"
+    run: touch src/Main.hx ;
+          echo "class Main {" >> src/Main.hx ;
+          echo "  static functon main() {" >> src/Main.hx ;
+          echo "    trace(\"hello, world\");" >> src/Main.hx ;
+          echo "  }" >> src/Main.hx ;
+          echo "}" >> src/Main.hx
```
Comment (BergerAPI) [inline templates/setupHaxeProject.yaml]: Please create a file in `./templates/src/` called `hello-world.hx` and paste its content in there. Your step can be replaced with `cp $PROJECT_DIR/templates/src/hello-world.hx ./src/Main.hx`

File: templates/setupHaxeProject.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - name: "Create Compile.hxml and its contents"
+    run: touch Compile.hxml ;
+          echo "--class-path src" >> Compile.hxml ;
+          echo "--main Main" >> Compile.hxml ;
+          echo "--hl bin/out.hl" >> Compile.hxml ;
+          echo "--cmd hl bin/out.hl" >> Compile.hxml
```
Comment (BergerAPI) [inline templates/setupHaxeProject.yaml]: Same as my first comment.

**Review by BergerAPI — CHANGES_REQUESTED** [review]

File: templates/setupHaxeProject.yaml (hunk 1 above)
Comment (IAmCheeseman) [reply templates/setupHaxeProject.yaml]: Fixed!

**Review by BergerAPI — APPROVED** [review]
Well done.

**Pull request merged**